### DISCUSSION TITLE
Modify the blog tags

### DISCUSSION THREAD

**Discussion by kev-odin** [alshedivat/al-folio#925] [Q&A] [answered]
Hello,

How do I change the default tags in the blog page?
The current defaults(formatting, images, links, math, code) do not reflect the content that I want to share.

Thank you!

**Comment by rohandebsarkar** [comment, marked as answer]
Change the following line in `_config.yml`:
https://github.com/alshedivat/al-folio/blob/fbbca1a8c4a29e5b9de70293068622069884f4f8/_config.yml#L225

**Reply by kev-odin** [reply]
Thank you!
